### PR TITLE
Add lending pool tvl for SFT Protocol

### PR DESCRIPTION
**Pull request by Sunnight-wj**
lending pool is a new pool that SFT Protoco users can borrow FIL from it by pledging SFT Fixed Order, so the borrowed amount is not counted under the active miner.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/sft-protocol exports TVL: 
        
 
 
    filecoin                  4.06 M
    
    total                    4.06 M

**Comment by g1nt0ki** [comment]
sorry for the delay, will review this soon

**Comment by g1nt0ki** [comment]
> hey guy, don't forget this PR @g1nt0ki

so, I looked at the docs: 
image

okay, I thought FIL borrowed will be used as part of liquid staking but if not, and it leaves the system, we can only count the amount of FIL in the pool not borrowed as your tvl,

the value of FIL lent out can be counted under 'borrowed' key, this is what we do for other lending protocols

**Comment by Sunnight-wj** [comment]
> 
I agree with you, I'll update it later.

**Comment by Sunnight-wj** [comment]
> > hey guy, don't forget this PR @g1nt0ki
> 
> so, I looked at the docs: image
> 
> okay, I thought FIL borrowed will be used as part of liquid staking but if not, and it leaves the system, we can only count the amount of FIL in the pool not borrowed as your tvl,
> 
> the value of FIL lent out can be counted under 'borrowed' key, this is what we do for other lending protocols

ok, I think you are right, I'll update it later.

**Comment by llamatester** [comment]
The adapter at projects/sft-protocol exports TVL: 
        
 
 
    filecoin                  3.85 M
    
    total                    3.85 M

**Comment by Sunnight-wj** [comment]
now only the lending pool available FIL is added to tvl.

File: projects/sft-protocol/index.js [hunk 1]
```diff
@@ -40,7 +44,11 @@ module.exports = {
     filecoin: {
         tvl: async (_, _1, _2, { api }) => {
 
-            let balances = {};
+            const totalPooledFILAmount = await api.call({ target: LENDING_POOL_ADDRESS, abi: GET_TOTAL_POOLED_FIL_ABI, params: [] });
+            const totalLendingFILAmount = await api.call({ target: LENDING_POOL_ADDRESS, abi: GET_TOTAL_LENDING_FIL_ABI, params: [] });
+            let lendingPoolTvl = totalPooledFILAmount - totalLendingFILAmount;
+            api.add(ADDRESSES.null, lendingPoolTvl);
+            let balances = await sumTokens2({ owner: LENDING_POOL_ADDRESS, tokens, api });
```
Comment (g1nt0ki) [inline projects/sft-protocol/index.js:51]: since we are fetching fil in this line and adding it to balances, line 50 and above is redundant?

**Comment by llamatester** [comment]
The adapter at projects/sft-protocol exports TVL: 
        
 
 
    filecoin                  3.69 M
    
    total                    3.69 M

File: projects/sft-protocol/index.js (hunk 1 above)
Comment (Sunnight-wj) [reply projects/sft-protocol/index.js:45]: my fault, the redundane code has been removed.

**Pull request merged**